api/prisma/seed-staging.ts

```diff
@@ -1365,4 +1385,21 @@ export const stagingSeed = async (
       password: 'abcdef',
     }),
   });
+  if (largeSeed) {
+    stagingRealisticAddresses.forEach(async (addr, index) => {
```

Comment:
I gotcha

the forEach loop still needs to be changed

Reply:
@YazeedLoonat Updated - but I thought this was the use case where it didn't matter? It also is working to seed all the listings. Just wanna make sure I understand!

Reply:
for sure! the issue here is that the seeding script will say that seeding has been completed when it hasn't 

its fine locally since we normally give it time between reseeding and starting the api, not necessarily by choice but just because we're slower than the computer processes these 

I think it could cause some weirdness where in a hosted env after the reseed the data isn't actually written yet and the db doesn't have enough connections for both prisma and the still connected and running seed script

Reply:
Thank was super helpful tysm!!